migrate/models/index.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const process = require('process');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.json')[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
```

Comment:
can we add a try-catch here, just to make sure, in case when database is unable to connect ?

Reply:
Also, logging a success message when database is connected is a good practice.

Reply:
Similarly, when database is unable to connect just log it, in the catch block.